api/src/context.rs

```diff
@@ -912,14 +912,15 @@ impl Context {
 
             let gas_schedule_params =
                 match GasScheduleV2::fetch_config(&storage_adapter).and_then(|gas_schedule| {
+                    let featrure_version = gas_schedule.feature_version;
```

Comment:
nit: typo featrure_version => feature_version

Reply:
Also nit: Why not just inline this if this var is only used once?

Reply:
> nit: typo featrure_version => feature_version

Thanks for catching this! I can't believe I made so many typos yesterday...

> Also nit: Why not just inline this if this var is only used once?

Tried that. Borrow checker's not happy.